packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/converter/convertHandles.ts

```diff
@@ -23,35 +23,43 @@ export const convertHandles = (gqlNode: GQLNode<GQLNodeStyle>, gqlEdges: GQLEdge
     if (edge.sourceId === gqlNode.id) {
       connectionHandles.push({
         id: `handle--source--${gqlNode.id}--${sourceHandlesCounter}`,
+        index: 0,
         nodeId: gqlNode.id,
         position: Position.Right,
         type: 'source',
+        virtual: edge.state === GQLViewModifier.Hidden,
```

Comment:
Maybe the actual flag we want is `hidden` instead of `virtual` (which is an ad-hoc concept).
Handles for edges would be `hidden` if and only if the corresponding edge is.
What we currently call "virtual" handles would be *always* hidden.

This does not change the actual behavior, so it's not required to make this change.
It's more of a remark.